frontend/src/components/ViewActivities/activityday.js

```diff
@@ -13,18 +11,12 @@ class ActivityDay extends React.Component {
     let date = new Date(this.props.date);
     let id = date.getTime();
     return (
-      <Card>
-        <Accordion.Toggle as={Card.Header} eventKey='0' align='center' >
-          {utils.timestampToDateFormatted(date.getTime())}
-        </Accordion.Toggle>
-        <Accordion.Collapse eventKey='0'>
-          <Card.Body>
-            {sortedActivities.map((activity, index) => (
-            <Activity activity={activity} key={index + id}/>
-            ))}
-          </Card.Body>
-        </Accordion.Collapse>
-      </Card>
+      <div className='activity-day'>
+        <h4>{time.timestampToDateFormatted(date.getTime())}</h4>
+        {sortedActivities.map((activity, index) => (
+          <Activity activity={activity} key={index + id} className="activity"/>
```

Comment:
I'm curious, why do you do `index + id` rather than just `index`? Won't the keys be unique in either scenario?

Reply:
There was some worry I had that `index` would be repeated across each instance of activityDay. I just double checked and turns out, that's not actually a problem. Fixed.